solution_template/nested/publicIP-new.json

```diff
@@ -4,7 +4,7 @@
   "outputs": {
     "fqdn": {
       "type": "string",
-      "value": "[reference(resourceId('Microsoft.Network/publicIPAddresses',parameters('publicIPName')),'2016-03-30').dnsSettings.fqdn]"
+      "value": "[reference(resourceId(parameters('publicIPResourceGroup'),'Microsoft.Network/publicIPAddresses',parameters('publicIPName')),'2016-03-30').dnsSettings.fqdn]"
```

Comment:
For new resources the resource group will be the same as for all the other resources. I left it there so  the output line can be the same in both templates.
I can remove it